### ISSUE TITLE
string Concat panic

### ISSUE THREAD

**Issue by zhangyc310** [Knetic/govaluate#13]
Hi,govaluate is superising , but when I concat string , it panic.
```
package main
import (
	"log"
	"github.com/Knetic/govaluate"
)
func main() {
	expression, err := govaluate.NewEvaluableExpression("'http_response_body' + 'service is ok'")
	if err != nil {
		log.Printf("err1: ", err)
	}
	parameters := make(map[string]interface{}, 8)
	parameters["http_response_body"] = "service is ok"
	result, err := expression.Evaluate(parameters)
	if err != nil {
		log.Printf("err2: ", err)
	}
	log.Printf("rs: ", result)
}
```

**Comment by Knetic** [comment]
Thanks for the report, and the sample!

It looks as if I left the implementation for string concatenation in a `git stash` but accidentally committed notes about it in the readme. You hit a bug with my process, it seems.

However, a fix has been pushed, so as of 63c7fee765ffe58e3c4693fd159702bb4c38d92d (which is in master branch now) your code sample ought to work.

**Comment by Knetic** [comment]
I can't repro anymore, so i'm closing this bug. Let me know if you still see problems.
